shared/test-helpers/ganache-cli.sh

```diff
@@ -15,10 +15,10 @@ testrpc_running() {
 
 start_testrpc() {
   if [ "$SOLIDITY_COVERAGE" = true ]; then
-    npx testrpc-sc -i 16 --gasLimit 0xfffffffffff --port "$testrpc_port"  > /dev/null &
```

Comment:
Mmmm, I'd like to keep this the same as in aragonOS; was there a reason for not starting the testrpc instance here? I'm not sure what flags `solidity-coverage` uses when it starts `testrpc-sc`.

Reply:
Yes, I was trying to follow these instructions: 
https://github.com/sc-forks/solidity-coverage/issues/131#issuecomment-417155288
So first I want to be able to make it work (no luck so far) and then I'll care about those details ;-) But, yes, I agree it would be better to keep like it was.